test/checkpoint/recovery-test.sh

```diff
@@ -2,27 +2,47 @@
 
 set -x
 
-home=${1:-WT_TEST}
+usage () {
+    cat << EOF
+Usage: recovery_test.sh {config} {loop_number_of_times}
+Arguments:
+    config    				# Configuration to the checkpoint binary
+    loop_number_of_times       # Number of times to run the test in a loop
+EOF
+}
+
+if [ "$#" -ne 2 ]; then
+    echo "Illegal number of parameters."
+    usage
+    echo FAILED
```

Comment:
Not sure this line adds much value. If it is could this line be moved up above usage.